components/identity.oauth.uma.permission.endpoint/src/gen/java/org/wso2/carbon/identity/oauth/uma/permission/endpoint/PermissionApi.java

```diff
@@ -1,15 +1,17 @@
 package org.wso2.carbon.identity.oauth.uma.permission.endpoint;
 
+import org.wso2.carbon.identity.oauth.uma.permission.endpoint.factories.PermissionApiServiceFactory;
+
 import io.swagger.annotations.ApiParam;
+
 import org.wso2.carbon.identity.oauth.uma.permission.endpoint.dto.PermissionTicketResponseDTO;
 import org.wso2.carbon.identity.oauth.uma.permission.endpoint.dto.ResourceModelDTO;
-import org.wso2.carbon.identity.oauth.uma.permission.endpoint.factories.PermissionApiServiceFactory;
 
-import javax.ws.rs.Consumes;
-import javax.ws.rs.POST;
-import javax.ws.rs.Path;
-import javax.ws.rs.Produces;
+import org.apache.cxf.jaxrs.ext.MessageContext;
+
 import javax.ws.rs.core.Response;
+import javax.ws.rs.*;
```

Comment:
Can't have import all. (*) Use wso2 code style to format the code.

Reply:
Since this is auto generated code is it okay to change it?

Reply:
Is it regenerating every time we compile ? If yes, ignore those files. Otherwise fix.

Reply:
No it is not regenerated